tiledb/sm/filter/bitshuffle_filter.cc

```diff
@@ -31,11 +31,13 @@
  */
 
 #include "tiledb/sm/filter/bitshuffle_filter.h"
-#include "bitshuffle_core.h"
 #include "tiledb/sm/enums/datatype.h"
+#include "tiledb/sm/enums/filter_type.h"
 #include "tiledb/sm/misc/logger.h"
 #include "tiledb/sm/tile/tile.h"
 
+#include "bitshuffle_core.h"
```

Comment:
include the full path to the header, e.g. `tiledb/sm/filter/bitshuffle_core.h`

Reply:
When I was cleaning up, I come across that a few header files are included like that and I fixed. However, I fail to notice this one. It will be fixed.

Reply:
BTW, there is no bitshuffle_core header file in filter folder. Currently, this file is located under **external/include/bitshuffle** folder.

Reply:
Oops -- thanks. This is fine.